jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -273,10 +273,10 @@ function configure_controller_access {
         # url
         local kieServerControllerUrl=$(build_simple_url "${kieServerControllerProtocol}" "${kieServerControllerHost}" "${kieServerControllerPort}" "${kieServerControllerPath}")
         JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.server.controller=${kieServerControllerUrl}"
+        # user/pwd
+        JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.server.controller.user=\"$(get_kie_server_controller_user)\""
+        JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.server.controller.pwd=\"$(esc_kie_server_controller_pwd)\""
     fi
-    # user/pwd
-    JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.server.controller.user=\"$(get_kie_server_controller_user)\""
-    JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.server.controller.pwd=\"$(esc_kie_server_controller_pwd)\""
     # token
     local kieServerControllerToken="$(get_kie_server_controller_token)"
```

Comment:
Shouldn't the token also get moved up alone with the user and pwd? It is just another way to authenticate - alternative to user/pwd) if a controller (host) is specified.

Reply:
It won't be necessary as 'get_kie_server_controller_token' function ensures a empty string, be returned if no correspondent environment variable were found. (See this)
https://github.com/rhtevan/jboss-kie-modules/blob/bbb38cc3ab149177162d286b48f3409bdd1fcc6b/jboss-kie-wildfly-common/added/launch/jboss-kie-wildfly-security.sh#L158-L161

Then, this line of code ensures no system property being set if value of the local variable is empty. 
https://github.com/rhtevan/jboss-kie-modules/blob/bbb38cc3ab149177162d286b48f3409bdd1fcc6b/jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh#L282

Reply:
@rhtevan True, but "token" is still a property for controller authentication just like "user" and "pwd", so it is best placed inside the same block as those properties.

Reply:
@errantepiphany Updated as per suggestion